### PR TITLE
add script to add tags to needles

### PR DESCRIPTION
**Pull request by coolo**
used to fix https://progress.opensuse.org/issues/9816, but might be useful
in the future for other cases

### CODE REVIEW

**Comment by okurz** [comment]
+1 though an example in usage wouldn't hurt

**Comment by coolo** [comment]
like where?

File: script/modify_needle [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+=item B<--add-tags>
+
+check the needle and add the given tags (comma separated) if not yet present
+
+=item B<--help, -h>
+
+print help
+
+=cut
```
Comment (okurz) [inline script/modify_needle]: somewhere up here

**Comment by coolo** [comment]
see a1522e8

**Pull request merged**